### ISSUE TITLE
Trigger Menu Open

### ISSUE THREAD

**Issue by aliaezat** [swisnl/jQuery-contextMenu#655]
Hi there,

I am trying to create jQuery-contextMenu on daypilot scheduler event (DayPilot.org) but the scheduler context menu is conflicting with it. Basically, daypilot is preventing the contextmenu event propagation. So, my question is, Is there a way I can get jQuery-contextMenu manually without explicitly triggering 'contextmenu' event like in using:

$(".some-selector").contextMenu();
$(".some-selector").contextMenu({x: 123, y: 123});

**Comment by bbrala** [comment]
Doesn't the code you provided work? That should open the contextmenu manually as [mentioned in docs](https://swisnl.github.io/jQuery-contextMenu/docs/plugin-commands.html)

**Comment by aliaezat** [comment]
It doesn't work because I have to call it from within a contextmenu event handler which causes an infinit recursion. I need away to open the context menu directly without triggering the 'contextmenu' event.

**Comment by russelltrafford** [comment]
Agreed, having the same issue here. Ends up with an infinite loop. In my case, setting a listener on contextmenu, and then conditionally deciding whether to open the menu or not, and if yes, the only way to trigger the menu is using contextmenu, which calls itself.
